x-pack/plugin/stack/src/main/java/org/elasticsearch/xpack/stack/StackTemplateRegistry.java

```diff
@@ -155,6 +158,12 @@ protected List<LifecyclePolicy> getPolicyConfigs() {
                 REGISTRY_VERSION,
                 TEMPLATE_VERSION_VARIABLE
             ),
+            new IndexTemplateConfig(
+                ECS_DYNAMIC_MAPPINGS_COMPONENT_TEMPLATE_NAME,
+                "/ecs-dynamic-mappings.json",
```

Comment:
Out of curiosity: I assume file names can change at any time without being a breaking change (just curious)

Reply:
Good, then we are aligned :-)